tests/tests.wake

```diff
@@ -202,11 +204,11 @@ def runTest (testScript: Path): Result Unit Error =
     def inTestDir path =
         relative testDirectory path
 
-    require Pass (Pair wakeBin wakeVisible) =
-        wakeToTest Unit
+    require Pass (Pair wakeDir wakeVisible) =
+        wakeToTestDir Unit
 
     def testJob =
-        makeExecPlan ("./{testScript.getPathName.inTestDir}", wakeBin.inTestDir, Nil) (visibleFiles ++ wakeVisible)
+        makeExecPlan ("./{testScript.getPathName.inTestDir}", wakeDir.inTestDir, Nil) (visibleFiles ++ wakeVisible)
         | setPlanDirectory testDirectory
         | setPlanLabel "testing: {testName}"
         | setPlanLocalOnly True # On OS/X you cannot mount fuse within fuse
```

Comment:
I guess this line here is saving us because otherwise this is problematic. The issue is that we don't have all of the generated binaries listed here. It works fine because this is running without fuse so all files are visible.

Ideally the test code would consume the binary paths from somewhere.

Maybe we want two interfaces 1) one that uses claimFileAsPath and a directory and 2) one that takes a list of visible binaries and extracts the directory from that list